scripts/verify-extract-api.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+changed_api_extracts=$(git status | grep -F API.md | wc -l)
+
+if [[ changed_api_extracts -gt 0 ]]; then
+  echo "Fail! Detected api change. Please run 'yarn extract-api' and add API.md file changes to the change set."
+  exit 1;
+else
+  echo "Success! No drift detected in API.md files."
+fi
```

Comment:
changes in this file.
- changed name to match `-` we use in other script names.
- added `-F` flag to make sure that grep doesn't interpret `.` in `API.md` as wildcard
- added else case to print successful message.